src/ucp/rndv/rndv.c

```diff
@@ -177,16 +178,23 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_proto_progress_rndv_rtr, (self),
     size_t packed_rkey_size;
     ucs_status_t status;
 
-    /* send the RTR. the pack_cb will pack all the necessary fields in the RTR */
+    /* Send the RTR, the pack_cb packs all the necessary fields in the RTR */
```

Comment:
Guess it meant two sentences separated by a period.

Reply:
fixed only starting with capital letter